fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -855,12 +856,29 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
         }
     }
 
-    private fun deviceRamBytes(): Long {
+    @VisibleForTesting
+    @Suppress("MagicNumber")
+    internal fun getDeviceAvailableMemoryInGB(): Long {
+        val memoryInfo = getMemoryInfo()
+        val gigabytesDivider = (1e+9).toLong()
```

Comment:
This calculation will give a number that's slightly off, right? Don't we want `1024 * 1024 * 1024 * 1024`?

Reply:
It was returning 7 for my pixel 6, which it's about the same that  it's showing on **Developer options** -> **Running services** :)

<img width="357" alt="image" src="https://github.com/mozilla-mobile/firefox-android/assets/773158/6d88fcc5-0b5c-46bc-9757-cdbb263362ab">

Anyway,  I think we will be moving to collect the raw data :)